src/libraries/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -383,5 +383,57 @@ internal static void AddEntry(ZipArchive archive, string name, string contents,
                 w.WriteLine(contents);
```

Comment:
This file should be in `tests` folder.

Reply:
Good point. I don't know why it was created there. Maybe we used to share something with another assembly, but it got removed.

Reply:
Ah search didn't return the other file that consumes ZipTestHelper. It's the ZipFile.csproj. I'll revert the move of this file.